### DISCUSSION TITLE
How to use hot reloading?

### DISCUSSION THREAD

**Discussion by onurusluca** [obytes/react-native-template-obytes#240] [Q&A] [answered]
I understand that we can't use expo go but how to get hot reloading on mobile?

> Because we’re using the Expo custom dev client to support native dependencies with the starter. The Expo Go app is not an option to consider here; instead, you need to create the app and install it on your simulator or device to start using it.

**Comment by yjose** [comment, marked as answer]
Hot reloading has nothing to do with Expo Go, @onurusluca. In our case, you need to generate the app first using `pnpm run ios`, and the app should be installed on your simulator. You can think of this app as your custom Expo Go. Then, running `pnpm run start` should work with hot reload the same way Expo Go work
